metadata-ingestion/src/datahub/ingestion/extractor/schema_util.py

```diff
@@ -52,6 +52,15 @@ def _get_column_type(field_type) -> SchemaFieldDataType:
     return dt
 
 
+def _is_nullable(field):
+    if isinstance(field.type, avro.schema.UnionSchema):
+        return field.type.schemas[0].name == "null"
```

Comment:
Can we check each schema in the union and if any of them == null the field should be nullable... You can technically have something like [ "long", "null" ] as far as I'm aware

Reply:
Ah you're right - seems like my previous comment was wrong.

For union types, they inherit default values from the first item, but nullability comes from any one of the union types being nullable.

Maybe something like this will work:
```suggestion
        return any(_is_nullable(schema) for schema in field.type.schemas)
```

Reply:
I made kind of this but not exactly. Would have to change method argument from Field to Schema to use exactly this but I think having the argument as a Field reads better.

Reply:
Yep the changes look good!